jest.config.js

```diff
@@ -0,0 +1,69 @@
+const { defaults } = require('jest-config')
+module.exports = {
+    // A set of global variables that need to be available in all test environments
+    globals: {},
+
+    // This option sets the URL for the jsdom environment. It is reflected in properties such as location.href
+    testURL: 'http://localhost:8000',
+
+    // Indicates whether the coverage information should be collected while executing the test
+    collectCoverage: false,
+
+    // An array of glob patterns indicating a set of files for which coverage information should be collected
+    collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**'],
```

Comment:
We probably want to exclude the `static` and `public` directories as well.

Reply:
I added them too. thanks @sean-binary